tsbridge/config.go

```diff
@@ -98,6 +102,8 @@ func NewConfig(ctx context.Context, opts *ConfigOptions) (*Config, error) {
 		return nil, fmt.Errorf("configuration file validation error: %s", err)
 	}
 
+	c.Storage = opts.StorageEngine
```

Comment:
Having a `storage` parameter in the configuration file that is always overwritten with the `STORAGE_ENGINE` environment variable is pretty confusing.

I'd suggest to choose one configuration mechanism (file or environment variable) and stick with it. If the long term plan is to allow ts-bridge to be deployed outside of App Engine, perhaps it's going to be simpler to move all options into the configuration file and get rid of environment variables completely.

Reply:
Agreed it is confusing. I'll work with the env variable only for now (removed it from the parsed struct) but I do plan to address the ENV vars wholesale at a later point. PTAL at the new implementation.